Rakefile

```diff
@@ -1,47 +1,93 @@
 require 'rake/testtask'
 
 task :default do
-    puts 'rake -T'
+  puts 'rake -T'
```

Comment:
It should be `` puts `rake -T` ``.
you can just copy and paste see the different.
(Notice that the quotation marks should be the one next to the keyboard number 1 not the one next to the keyboard enter.)